avocado/plugins/resolver_avocado_instrumented.py

```diff
@@ -0,0 +1,52 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2019
+# Authors: Cleber Rosa <crosa@redhat.com>
+
+"""
+Test resolver for Avocado Instrumented tests
+"""
+
+import os
+
+from avocado.core.plugin_interfaces import Resolver
+from avocado.core.safeloader import find_avocado_tests
+from avocado.core.resolver import ReferenceResolution
+from avocado.core.resolver import ReferenceResolutionResult
+from avocado.core.nrunner import Runnable
+
+
+class AvocadoInstrumentedResolver(Resolver):
```

Comment:
I think the `AvocadoInstrumentedResolver`, `ExecTestResolver` and `PythonUnittestResolver` could be inside one file.  Or is there any reason to have them split into their files?

Reply:
That's a good point.  I'll do that in the next version.